### PR TITLE
Translate Dispatcher Filters

### PR DESCRIPTION
**Pull request by tavaresgerson**

### CODE REVIEW

File: pt/development/dispatch-filters.rst [hunk 1]
```diff
@@ -1,12 +1,199 @@
...
+
+Quanto maior a prioridade, mais tarde esse filtro será chamado.
+
+Aplicação condicional de filtros
+--------------------------------
+
+Se você não deseja executar um filtro em todas as solicitações, poderá usar condições 
+para aplicá-lo apenas algumas vezes. Você pode aplicar condições usando as opções ``for`` 
+e ``when``. A opção ``for`` permite que você combine com substrings de URL, enquanto a 
+opção ``when`` permite executar uma chamada:
```
Comment (markstory) [inline pt/development/dispatch-filters.rst]: Missing a second : at the end of this line.

File: pt/development/sessions.rst [hunk 2]
```diff
@@ -1,17 +1,424 @@
...
+        'ini' => [
+            'session.cookie_path' => '/',
+            'session.cookie_domain' => '.yourdomain.com'
+        ]
+    ]);
+
+By default PHP sets the session cookie to expire as soon as the browser is
+closed, regardless of the configured ``Session.timeout`` value. The cookie
+timeout is controlled by the ``session.cookie_lifetime`` ini value and can be
+configured using::
```
Comment (markstory) [inline pt/development/sessions.rst]: Is a code block missing here?

File: pt/development/sessions.rst [hunk 3]
```diff
@@ -1,17 +1,424 @@
...
+
+Por padrão, o PHP define o cookie da sessão para expirar assim que o navegador é fechado, 
+independentemente do valor configurado ``Session.timeout``. O tempo limite do cookie é 
+controlado pelo valor ini ``session.cookie_lifetime`` e pode ser configurado usando:
```
Comment (markstory) [inline pt/development/sessions.rst]: Need a second : at the end of this line.

File: pt/development/sessions.rst [hunk 4]
```diff
@@ -1,17 +1,424 @@
...
 .. note::
-    Atualmente, a documentação desta página não é suportada em português.
+    Antes da versão 3.6.0, os arquivos do adaptador de sessão devem ser colocados em
+    **src/Network/Session/AppHandler.php**.
+
+
+Sessões de Banco de Dados
+-------------------------
+
+Se você precisar usar um banco de dados para armazenar os dados da sessão, configure da seguinte maneira:
```
Comment (markstory) [inline pt/development/sessions.rst]: Need a second : at the end of this line.

File: pt/development/sessions.rst (hunk 3 above)
Comment (tavaresgerson) [reply pt/development/sessions.rst]: Ops! I'ts not ready

File: pt/development/sessions.rst (hunk 2 above)
Comment (tavaresgerson) [reply pt/development/sessions.rst]: Done

File: pt/development/sessions.rst (hunk 4 above)
Comment (tavaresgerson) [reply pt/development/sessions.rst]: Ops! I'ts not ready

File: pt/development/dispatch-filters.rst (hunk 1 above)
Comment (tavaresgerson) [reply pt/development/dispatch-filters.rst]: Done

**Pull request merged**